onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -173,29 +173,33 @@ bool QNNExecutionProvider::IsNodeSupported(qnn::QnnModelWrapper& qnn_model_wrapp
       // For Cast, need to call IsOpSupported (below) to validate input and output types.
       // For other single non-qdq nodes, immediately return not supported.
       if (node_unit.OpType() != "Cast") {
-        LOGS(logger, VERBOSE) << "Non-QDQ single node is not supported for NPU backend. Node name: " << node_unit.Name()
-                              << " Op type: " << node_unit.OpType();
+        LOGS(logger, WARNING) << "Non-QDQ " << node_unit.OpType()
+                              << " operators are not supported on HTP or DSP backends. " << node_unit.OpType()
+                              << " node `" << node_unit.Name() << " will not be assigned to QNN EP.";
```

Comment:
I upgraded log messages for unsupported ops to WARNING, so that they're easier to spot. There's a lot of VERBOSE output. I also made sure that both the op name and type are printed out.